src/ElasticsearchDAO.py

```diff
@@ -42,28 +42,28 @@ def writeToElasticsearch(self, datasets):
         else:
             print('Elasticsearch already up to date.')
 
-    def mapObjDoc(self, dataset, lineobjDoc):
-        newLineobjDoc = {}
-        newLineobjDoc['id'] = dataset.id
-        newLineobjDoc['name'] = dataset.name
-        newLineobjDoc['description'] = self.cleanText(dataset.description)
-        newLineobjDoc['accessLevel'] = dataset.access_level
-        newLineobjDoc['lastUpdate'] = dataset.last_updated
-        newLineobjDoc['tags'] = dataset.tags
-        newLineobjDoc['sourceUrl'] = dataset.source_url
-        newLineobjDoc['metrics'] = dataset.metrics
-        newLineobjDoc['doi'] = dataset.doi
-        newLineobjDoc['dhId'] = dataset.dh_id
-        newLineobjDoc['dhLastUpdate'] = dataset.dh_last_updated
-        newLineobjDoc['dhSourceName'] = dataset.dh_source_name
-        newLineobjDoc['dhType'] = dataset.dh_type
-        newLineobjDoc['dhProjects'] = lineobjDoc['_source']['dhProjects']\
-            if ('_source' in lineobjDoc and 'dhProjects' in lineobjDoc['_source']) else []
-        newLineobjDoc['dhDataTypes'] = lineobjDoc['_source']['dhDataTypes']\
-            if ('_source' in lineobjDoc and 'dhDataTypes' in lineobjDoc['_source']) else []
-        return newLineobjDoc
+    def map_obj_doc(self, dataset, line_obj_Doc):
```

Comment:
Good work - this is looking a lot better! Remember to change `line_obj_Doc` to `line_obj_doc` (all lowercase) so that the variable name will satisfy the code smell you're trying to fix.